docs/ManualMocks.md

```diff
@@ -8,15 +8,30 @@ instead of accessing a remote resource like a website or a database, you might
 want to create a manual mock that allows you to use fake data. This ensures your
 tests will be fast and not flaky.
 
+### Mocking internal modules
```

Comment:
`Mocking user modules`? Internal is ambiguous

Reply:
Right!